modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -2186,7 +2186,7 @@ void GDScriptAnalyzer::reduce_identifier_from_base(GDScriptParser::IdentifierNod
 		} else {
 			switch (base.builtin_type) {
 				case Variant::NIL: {
-					push_error(vformat(R"(Invalid get index "%s" on base Nil)", name), p_identifier);
+					push_error(vformat(R"(Value of type Nil does not have the attribute "%s".)", name), p_identifier);
```

Comment:
Can we have information about what identifier is of type Nil? Or would that be explicit enough from the stacktrace?

Reply:
Wonder why the keyword is `null` and its name is `nil`. They have different meanings in some other languages (like Objective C, Scala). In general, NULL is 0 or `'\0'` in ASCII and Nil is emptiness. (I'd like it as `null`)

Reply:
We discussed this in a GDScript meeting and suggest indeed referring to `null` directly. Maybe this could be:
```suggestion
					push_error(vformat(R"(Could not get attribute "%s" on a null value.)", name), p_identifier);
```

Reply:
Or the same phrasing as used in your VM changes to be consistent.

Reply:
I mean the sentence you used for the null case in `gdscript_vm.cpp`:
```
err_text = "Tried to access attribute " + v + " on a value that is null.";
```